app/src/test/java/org/gdg/frisbee/android/chapter/ChapterComparatorTest.java

```diff
@@ -0,0 +1,117 @@
+package org.gdg.frisbee.android.chapter;
+
+import android.location.Location;
+import android.location.LocationManager;
+
+import org.gdg.frisbee.android.BuildConfig;
+import org.gdg.frisbee.android.api.model.Chapter;
+import org.gdg.frisbee.android.api.model.Geo;
+import org.gdg.frisbee.android.app.BaseApp;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.junit.Assert.assertTrue;
+
+@RunWith(RobolectricGradleTestRunner.class)
+@Config(constants = BuildConfig.class, application = BaseApp.class)
+public class ChapterComparatorTest {
+
+    private Location locationIstanbul;
+
+    private static final Chapter chapterWithoutLocation = new Chapter("GDG AAUA", "115737274430015604080");
+    private static final Chapter chapterStartsWithLetterA = new Chapter("GDG AAUA", "115737274430015604080");
+    private static final Chapter chapterStartsWithDigit = new Chapter("GDG 6th October", "105609152381762460369");
+    private static final Chapter chapterIstanbul = new Chapter("GDG Istanbul", "100514812580249787371");
+    private static final Chapter chapterEskisehir = new Chapter("GDG Eskişehir", "115602346615910585045");
+    private static final Chapter chapterWithDoubleSpaceInName = new Chapter("GDG  Mukono", "108217555392256442794");
+
+    @Before
+    public void setup() {
+        locationIstanbul = new Location(LocationManager.GPS_PROVIDER);
+        locationIstanbul.setLatitude(28.97702);
+        locationIstanbul.setLongitude(41.011511);
+
+        chapterStartsWithLetterA.setGeo(new Geo(7.083333, 4.833333));
+        chapterStartsWithDigit.setGeo(new Geo(29.938126, 30.91398));
+        chapterIstanbul.setGeo(new Geo(28.97696, 41.00527));
+        chapterEskisehir.setGeo(new Geo(39.7766667, 30.5205556));
+    }
+
+    @Test
+    public void shouldBeAlphabeticalWhenNoHomeAndLocation() {
+        ChapterComparator comparator = new ChapterComparator(null, null);
+
+        //Chapter starting with digit should come before chapter starting with 'A'
+        assertTrue(0 > comparator.compare(chapterStartsWithDigit, chapterStartsWithLetterA));
```

Comment:
Definitely, thank you for the suggestion. It was in my mind but got list at the end.